client-v2/src/services/faciaApi.ts

```diff
@@ -100,6 +100,27 @@ async function fetchVisibleArticles(
   }
 }
 
+async function discardDraftChangesToCollection(
+  collectionId: string
+): Promise<void> {
+  // The server does not respond with JSON
```

Comment:
should we make it?

Reply:
This endpoint is also used by v1 so unless we make a whole different api for v2 I'm not too keen to start changing it.